src/org/sipdroid/sipua/SipdroidEngine.java

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (C) 2016 Pranav Jain - The NorthCap University - India

```

Comment:
put yourself below the other copyright names, also see my earlier comment about whether to mention your university